src/frontend/staff-portal/src/app/components/staff-workbench/contact-info/contact-info.component.ts

```diff
@@ -184,7 +184,24 @@ export class ContactInfoComponent implements OnInit {
   }
 
   public onSubmit(): void {
-    this.putDispute({ ...this.lastUpdatedDispute, ...this.form.value });
+    this.lastUpdatedDispute.disputantSurname = this.form.get('disputantSurname').value;
+    this.lastUpdatedDispute.disputantGivenNames = this.form.get('disputantGivenNames').value;
+    this.lastUpdatedDispute.driversLicenceNumber = this.form.get('driversLicenceNumber').value;
+    this.lastUpdatedDispute.driversLicenceProvince = this.form.get('driversLicenceProvince').value;
+    this.lastUpdatedDispute.driversLicenceIssuedCountryId = this.form.get('driversLicenceCountryId').value;
+    this.lastUpdatedDispute.driversLicenceIssuedProvinceSeqNo = this.form.get('driversLicenceProvinceSeqNo').value;
```

Comment:
driversLicenceIssuedCountryId
driversLicenceCountryId

driversLicenceIssuedProvinceSeqNo
driversLicenceProvinceSeqNo

Reply:
...ah i see its the different names between lastUpdatedDispute and the form controls